sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

```diff
@@ -347,3 +348,41 @@ Then the maximum memory bandwidth can be obtained using the standard get\_info()
     if (dev.has(aspect::ext_intel_max_mem_bandwidth)) {
       auto maxBW = dev.get_info<info::device::ext_intel_max_mem_bandwidth>();
     }
+
+# Free Memory #
+
+A new device descriptor will be added which will provide the number of bytes of free memory for the device.
```

Comment:
Maybe a little pedantic, but should this specify that it is global memory?

Reply:
OK